pkg/metrics/metrics.go

```diff
@@ -771,13 +777,19 @@ func NewLegacyMetrics() *LegacyMetrics {
 			Help:       "Highest policy revision number in the agent",
 		}),
 
-		PolicyChangeTotal: metric.NewCounterVec(metric.CounterOpts{
+		PolicyChangeTotal: metric.NewCounterVecWithLabels(metric.CounterOpts{
 			ConfigName: Namespace + "_policy_change_total",
 
 			Namespace: Namespace,
 			Name:      "policy_change_total",
 			Help:      "Number of policy changes by outcome",
-		}, []string{"outcome"}),
+		}, metric.Labels{
+			{
+				Name: "outcome",
+				// TODO: Use common label values for success/fail(ure) across all metrics.
+				Values: metric.NewValues("success", "failure"),
```

Comment:
Perhaps define SuccessValues, TruthValues, UpdateValues, etc... to normalize  and reuse?

Reply:
Success appears to already exist, so I'll use that.  "failure" is particular to this metric so I'll put it in a const but make a note that this is for legacy use.

In the future we should encourage the use of just "fail" and we should eventually follow up with cleaning up these labels and documenting the changes.